src/js/popup/app.js

```diff
@@ -234,15 +234,15 @@ define("popup/app", [
 					event = "unsuspend";
 				}
 
-					// switch to the selected tab
-				chrome.tabs.update(tab.id, updateData);
-
-					// make sure that tab's window comes forward
+				// make sure that tab's window comes forward
 				if (!isNaN(tab.windowId) &&
```

Comment:
I think we can get rid of this `if` entirely, and just put the `windows.update()` on a line right before the `tabs.update()`.  I'm not sure why I was checking the `windowId`.  It should always be available.